docs/user/README.md

```diff
@@ -0,0 +1,63 @@
+
+In order to implement Day-2 operations for a distributed application running in a container runtime, the single components of an application need to expose these signals by employing modern instrumentation. Furthermore, the signals need to get collected and enriched with the infrastructural metadata in order to ship them to a target system.
+
+![Stages of Observability](./assets/general-stages.drawio.svg)
+
+There are plenty of observability backends available either as a service or as a self-manageable solution focusing on different aspects and scenarios. Here, one solution will never fit all sizes, and the need to integrate with a specific solution will always be present. That's why instrumenting and shipping your telemetry data easily in a vendor-neutral way is relevant for Kyma to enable observability for your application with a low effort by integrating it into existing backends. These aspects must happen alongside your application, and managed tooling with guidance can benefit users' initial investment and maintenance effort. Also, Kyma will not focus on providing a managed in-cluster backend solution as a solution for an enterprise-grade setup will demand a central out-cluster solution.
+
+The Telemetry module focuses exactly on these aspects (instrumentation/collection/shipment) happening in the runtime and explicitly defocuses on backends. In addition, you can check the [tutorials](05-tutorials.md) on how to install lightweight in-cluster backends for demo or development purposes.
+
+## Features
+
+The Telemetry module enables your application with the telemetry support by providing the following:
+
+- guidance for the instrumentation - based on [Open Telemetry](https://opentelemetry.io/), community samples are provided on how to instrument your code using the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/) in nearly any programming language
+- tooling for collection, filtering, and shipment - based on the [Open Telemetry Collector](https://opentelemetry.io/docs/collector/), you can configure basic pipelines to filter and ship telemetry data
+- integration in a vendor-neutral way to a vendor-specific observability system - based on the [OpenTelemetry protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/), backend systems can be integrated
+- opt-out from features for advanced scenarios - at any time you can opt out per data type and execute the telemetry data collection/shipment with custom tooling
+- SAP BTP as first-class integration - integration into BTP Observability services will be prioritized
+- Enterprise-grade qualities - the setup is battle-tested and will satisfy typical development standards
```

Comment:
```suggestion
- Enterprise-grade qualities: The setup is battle-tested and satisfies typical development standards.
```

This sounds like marketing-blurb phrases. What exactly does this mean?? 
Shouldn't it go without saying that we satisfy typical dev standards? That's kind of a low bar?